backend/typescript/services/interfaces/IReviewService.ts

```diff
@@ -24,6 +24,10 @@ export interface PublisherResponse {
   publishYear: number;
 }
 
+export interface Genre {
```

Comment:
Do we need an explicit `Genre` interface? Since we're only displaying Genre names, we could just use `string[]` everywhere instead of `Genre[]`. 
The reviews API response would look something like:
```json
{
  "id": 1,
  "books": [
    "id": 1,
    ...
    "genres": ["a", "b", "c"], // strings here
  ],
  ...
}

Reply:
EDIT: probably too much work to